### ISSUE TITLE
[>=1.5.0] Docker pipeline unable to start / is stalled.

### ISSUE THREAD

**Issue by Mayzie** [drone/drone#2850]
Using Drone with the command `docker run --volume=/var/run/docker.sock:/var/run/docker.sock --volume=/var/lib/drone:/data --env-file=./drone.env --publish=80:80 --publish=443:443 --restart=always --name=drone drone/drone:1.5.0` causes all new builds to stall / hang and not even start when pushing a new commit (from GitHub). Using the UI and going into a stalled build, it just shows the name of the pipeline, and does not show the individual steps of the pipeline like it used to.

Downgrading to 1.4.0 allows pipelines to run as normal when pushing a new commit.

**Comment by bradrydzewski** [comment]
:wave: hey there, per our [issue template](https://raw.githubusercontent.com/drone/drone/master/.github/issue_template.md), for end-user support please use one of our support channels, such as our [chatroom](https://gitter.im/drone/drone) or our [mailing list](https://discourse.drone.io).

**Comment by bradrydzewski** [comment]
this was covered in the [release notes](https://discourse.drone.io/t/1-5-0-release-notes/5797)

**Comment by Mayzie** [comment]
It was my mistake, I jumped the gun when creating a new issue. I decided to randomly check the support forums and came across 1.5.0 release notes, which mention that multi-machine is now default, and to add the Drone environment variable `DRONE_AGENTS_DISABLED=true`. I normally just check the `CHANGELOG.md` file on git which didn't mention anything as that's generally more up to date, as some releases aren't documented in the forums.
